lib/pages/login-page.js

```diff
@@ -42,6 +42,36 @@ export default class LoginPage extends BaseContainer {
 		} );
 	}
 
+	use2FAMethod( twoFAMethod ) {
+		let actionSelector;
+
+		if ( twoFAMethod === 'sms' ) {
+			actionSelector = By.linkText( 'Code via text message' );
```

Comment:
These selectors will have a problem if/when we run these tests in other languages.  Can we add a data attribute in wp-calypso like [this](https://github.com/Automattic/wp-calypso/pull/21825) instead?

If that's not a viable option, we can leave this as-is for now.  But I'd recommend writing an issue in the wp-e2e-tests repo tagged with `workaround` just to document that these strings are here and a better solution should be found at some point.

Reply:
👍 I'll create an issue as soon as it is merged!